src/modules/i18n/locales/rm.json

```diff
@@ -81,7 +81,8 @@
 	"copy_success": "Copià il link!",
 	"copy_url": "Copiar il link",
 	"copyright_data": "© Datas:",
-	"copyright_label": "Copyright & decleraziun da protecziun da datas",
+	"copyright_label": "Nutzungsbedingungen",
+	"copyright_url": "https://www.geo.admin.ch/de/allgemeine-nutzungsbedingungen-bgdi/",
```

Comment:
I see that the page for the romanche in CMS is missing, does the CMS webmaster knows about this and will add it ?

Reply:
I don't know what's the stand with romanche, but for now if you try to access any page with this language it says that there is no translation yet. (https://www.geo.admin.ch/rm/)

Reply:
I know but I would ask if this is know and if it is planned to be fixed, if it is then I would use the correct path here even if it is a not found page, otherwise it will be forgotten.